[2024-09-11T06:34:27.296+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T06:34:27.312+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T06:34:25.042754+00:00 [queued]>
[2024-09-11T06:34:27.322+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T06:34:25.042754+00:00 [queued]>
[2024-09-11T06:34:27.322+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-11T06:34:27.339+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): query_data_from_mongo> on 2024-09-11 06:34:25.042754+00:00
[2024-09-11T06:34:27.349+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12852) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T06:34:27.350+0000] {standard_task_runner.py:72} INFO - Started process 12853 to run task
[2024-09-11T06:34:27.349+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Filter_data', 'query_data_from_mongo', 'manual__2024-09-11T06:34:25.042754+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/data-prep-for-model.py', '--cfg-path', '/tmp/tmp67nxqppn']
[2024-09-11T06:34:27.352+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask query_data_from_mongo
[2024-09-11T06:34:27.395+0000] {task_command.py:467} INFO - Running <TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T06:34:25.042754+00:00 [running]> on host dca6f81c3bd4
[2024-09-11T06:34:27.479+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Filter_data' AIRFLOW_CTX_TASK_ID='query_data_from_mongo' AIRFLOW_CTX_EXECUTION_DATE='2024-09-11T06:34:25.042754+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-11T06:34:25.042754+00:00'
[2024-09-11T06:34:27.480+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T06:34:27.583+0000] {transport.py:1923} INFO - Connected (version 2.0, client OpenSSH_8.2p1)
[2024-09-11T06:34:27.640+0000] {transport.py:1923} INFO - Authentication (password) successful!
[2024-09-11T06:34:32.418+0000] {python.py:240} INFO - Done. Returned value was: /opt/***/data/mongo_data.json
[2024-09-11T06:34:32.443+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T06:34:32.444+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Filter_data, task_id=query_data_from_mongo, run_id=manual__2024-09-11T06:34:25.042754+00:00, execution_date=20240911T063425, start_date=20240911T063427, end_date=20240911T063432
[2024-09-11T06:34:32.478+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-11T06:34:32.515+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-11T06:34:32.517+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
